mina.netty/src/test/java/org/kaazing/mina/netty/NioDatagramChannelIoAcceptorIT.java

```diff
@@ -56,16 +55,20 @@
     @Rule
     public TestRule timeout = new DisableOnDebug(new Timeout(10, TimeUnit.SECONDS));
 
-    private IoAcceptor acceptor;
+    private NioDatagramChannelIoAcceptor acceptor;
     private DatagramSocket socket;
 
     @Before
     public void initResources() throws Exception {
-        DatagramChannelIoSessionConfig sessionConfig = new DefaultDatagramChannelIoSessionConfig();
+        DatagramChannelIoSessionConfig sessionConfig = new DefaultNioDatagramChannelIoSessionConfig();
         sessionConfig.setReuseAddress(true);
+        sessionConfig.setReceiveBufferSize(65536);
+        sessionConfig.setSendBufferSize(65536);
         NioWorkerPool workerPool = new NioWorkerPool(newCachedThreadPool(), 4);
         NioServerDatagramChannelFactory channelFactory = new NioServerDatagramChannelFactory(newCachedThreadPool(), 1, workerPool);
         acceptor = new NioDatagramChannelIoAcceptor(sessionConfig, channelFactory);
+        acceptor.setReceiveBufferSize("65536");
```

Comment:
acceptor changes are for parent channel which is handled by boss.
sessionConfig changes for child channels (not really needed as they are not real datagram channels) and client channels. 

We don't have this for TCP as all the channels are handled by sessionConfig. But there is some config like setReuseAddress is set on acceptor and sessionConfig in tcp.